core/src/main/java/io/camunda/migrator/HistoryMigrator.java

```diff
@@ -139,18 +139,9 @@ public void start() {
   }
 
   private void printSkippedHistoryEntities() {
-    List<IdKeyMapper.TYPE> historyTypes = List.of(
-        HISTORY_PROCESS_DEFINITION,
-        HISTORY_PROCESS_INSTANCE,
-        HISTORY_FLOW_NODE,
-        HISTORY_USER_TASK,
-        HISTORY_VARIABLE,
-        HISTORY_INCIDENT,
-        HISTORY_DECISION_DEFINITION,
-        HISTORY_DECISION_INSTANCE
-    );
-
-    historyTypes.forEach(this::printSkippedEntitiesForType);
+    Arrays.stream(IdKeyMapper.TYPE.values())
+        .filter(type -> type != IdKeyMapper.TYPE.RUNTIME_PROCESS_INSTANCE)
```

Comment:
Maybe inclusive filtering is better than exclusive (for example if we add new enums that are not history, we might overlook it and/or need refactoring).

```suggestion
        .filter(type -> type.name().startsWith("HISTORY"))
```